bot/exts/info/information.py

```diff
@@ -547,7 +547,21 @@ async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
         for rule in rules:
             self.bot.stats.incr(f"rule_uses.{rule}")
 
-        final_rules = tuple(f"**{pick}.** {full_rules[pick - 1]}" for pick in rules)
+        if rules:
+            final_rules = tuple(f"**{pick}.** {full_rules[pick - 1][0]}" for pick in rules)
+        else:
+            final_rules = tuple(f"**{pick + 1}** {full_rules[pick][0]}" for pick, rule in enumerate(full_rules)
+                                if keyword in rule[1])
+
+        if not rules and not final_rules:
+            # This would mean that we didn't find a match for the used keyword
+            rules_embed.description = (
+                f"There are currently no rules that correspond to keyword: {keyword}."
+                " If you think it should be added, please ask our admins and they'll take care of the rest."
```

Comment:
This seems too trivial of a matter to require admin attention. I think it should direct users to one of `#dev-contrib`, `#meta`, or creating an issue on GH. @python-discord/admins thoughts?

Reply:
I agree, I just wanted to put it there as placeholder at first, and the response if it was fitting. But definitely too trivial of a matter.

Reply:
Yea, I think raising it in #community-meta makes sense here. Allows staff that don't watch #dev-contrib to give opinions too.

Reply:
Here's a preview
![image](https://user-images.githubusercontent.com/48383734/189521687-68479753-860c-45cf-84a5-d6102a03a15a.png)